sentry-okhttp/src/main/java/io/sentry/okhttp/SentryOkHttpEventListener.kt

```diff
@@ -406,6 +406,7 @@ public open class SentryOkHttpEventListener(
 
     private fun canCreateEventSpan(): Boolean {
         // If the wrapped EventListener is ours, we shouldn't create spans, as the originalEventListener already did it
-        return originalEventListener !is SentryOkHttpEventListener
+        return originalEventListener !is SentryOkHttpEventListener &&
+            "io.sentry.android.okhttp.SentryOkHttpEventListener" != originalEventListener?.javaClass?.name
```

Comment:
`l`: any useful comment we can add here as to why the additional check is necessary?